src/nrniv/CMakeLists.txt

```diff
@@ -208,6 +208,7 @@ set(NRN_INCLUDE_DIRS
     ${NRN_OC_SRC_DIR}
     ${PROJECT_BINARY_DIR}
     ${PROJECT_BINARY_DIR}/src/nrncvode
+    ${PROJECT_BINARY_DIR}/src/nrnmpi
```

Comment:
I have no real objection to the generic inclusion of this. Just wondering if there is any compilation time cost given that the files in this folder are included only by files in src/nrnmpi.

Reply:
From local testing, I'm getting a negligible difference in build times compared to `master`. I think it makes sense, since the cost of listing the files themselves (there is only a handful of them anyway) is very small compared to the preprocessing/compilation steps.